addons/music/stringtable.xml

```diff
@@ -3,9 +3,11 @@
     <Package name="Music">
         <Key ID="STR_ZEN_Music_ThisIsWarPartOne">
             <English>This Is War - Part 1</English>
+            <Polish>To jest Wojna - Część 1</Polish>
         </Key>
         <Key ID="STR_ZEN_Music_ThisIsWarPartTwo">
             <English>This Is War - Part 2</English>
+            <Polish>To jest Wojna - Część 2</Polish>
```

Comment:
These are music track titles, not sure if they should be translated.

Reply:
True

Reply:
Marked as resolved but music title translations are still there.